src/commands/converter/addressToHex.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ *
+ * Copyright 2018 NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import chalk from 'chalk';
+import {command, ExpectedError, metadata, option} from 'clime';
+import {Convert, RawAddress} from 'nem2-sdk';
+import {OptionsResolver} from '../../options-resolver';
+import {ProfileCommand, ProfileOptions} from '../../profile.command';
+
+export class CommandOptions extends ProfileOptions {
+    @option({
+        flag: 'a',
+        description: 'Address address',
+    })
+    address: string;
+
+    validateAddress(value: string) {
+        if (RawAddress.constants.sizes.addressEncoded !== value.length) {
+            throw new ExpectedError(`${value} does not represent a valid base32 address`);
+        }
+        return value;
+    }
+}
+
+@command({
+    description: 'Converts a base32 address to a hex address',
```

Comment:
should use `encoded address` and  `decoded address`

Reply:
Hello, I think these words are too professional. When users first use the method, they may not know what to fill out.

Reply:
my personal view is encoded is better than base32 ( as used in the sdk), @dgarcia360 ?

Reply:
@rg911 I proposed this naming, but it can make sense to use encoded and decoded instead to have consistency between projects.

Let me know if the following equivalence is correct:

encoded =   base32 = SDZH2C-7P755M-V6QPTE-DVMFKS-OT77FT-DIIKOQ-52YG
decoded = hex = C51BC5F3D7F9E1758F9B8B6D250BE4EB1B56877330990655C1DEBEE04DDA80E1

or if it's the other way round.

Reply:
careful here, I believe in catapult-server its the other way around. We should take this decision with `encoded` meaning from catapult-server (which is HEX iirc).

Reply:
https://github.com/nemtech/catapult-server/blob/95e0c6a1d700aedb9ca60faaf2b535f587012d7f/tests/catapult/model/AddressTests.cpp#L34

Reply:
Catapult-server seems to be fine. Please check if there is an inconsistency in the SDK/REST.

Reply:
hmm. didn't realize that as I thought the same as @evias stated.

Reply:
OK. Sorry and thanks @dgarcia360, I was wrong then. :+1:

`encoded` = **base32**
`decoded` = **hexadecimal**

Reply:
@NemTechCN Sorry for the confusion, the specification was defined using ``base32``and ``hex``. 

This is the summary of the changes we could apply to make it more consistent with other layers:

**File names**

``addressToHex`` -> **decodeAddress**
``hexToAddress``-> **encodeAddress**

**Variables**

base32 -> **encoded**
hexadecimal -> **decoded**